### ISSUE TITLE
UserView GET response content

### ISSUE THREAD

**Issue by jonnybazookatone** [adsabs/biblib-service#30]
@aholachek:
Information received from a GET on the user view: right now it seems to return
{libraries : [{name: lib_name, id: lib_id, description: lib_description}]}.

What would be the feasibility of adding the following keys to this summary data:  
{permission :[read/write/admin], num_papers : int, date_created: date, date_last_modified: date}  ?

**Comment by jonnybazookatone** [comment]
For the dates, for now I would guess they would take the datetime object form from python:
```
>>> datetime.datetime.now().isoformat()
'2015-06-03T10:06:47.582143'
```

The library ID is a base64 string representation of the UUID of the underlying library, e.g.
```
>>> library_id = uuid.uuid4()
>>> library_id_url = base64.urlsafe_b64encode(x.bytes)
>>> print library_id
UUID('a346d4c0-d1c1-4757-a8d0-b550e8a61d04')
>>> print library_id_url
'o0bUwNHBR1eo0LVQ6KYdBA=='
```
Currently, for prettifying, the trailing '==' are removed and any slashes are replaced by underscores.

For the permissions I haven't decided exactly. I am on the fence at the moment. If I put them in the GET end point of the /libraries then I think this would make redundant, or question, the need for a /permissions GET end point (issue #31).

Regardless, the output can be of two forms depending on the desired output. permission = <single value> or permissions = [<list of values>], where the values can be 'read', 'write', 'owner', and/or 'admin'. There's currently no logic implemented to force there to be a single type of permission, so in principle they could have read+write (this would not affect the permission to carry out something). This is indeed redundant, and will be altered at some stage, so I guess the single value makes more sense.

**Comment by jonnybazookatone** [comment]
Display e-mail of the owner?

**Comment by jonnybazookatone** [comment]
Merged #45:

library_status : "public"/ "private, only you have access" / "private, 12 people have access" that I can display on the library detail page? Unless people with read/write access should not know how many other people have access to the library?

owner : the email of the owner of the library

**Comment by jonnybazookatone** [comment]
The owner details will be decided and implemented in issue #46.
